src/ConfigResource.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
I don't think we can use `declare(strict_types=1)` in a minor release: too risky, since a lot of type juggling is involved in this package.

Reply:
Hi @Ocramius, the `declare(strict_types=1)` additions were added automatically by running `composer cs-fix` (`phpcbf`), which applies the [LaminasCodingStandard](https://github.com/laminas/laminas-coding-standard) PHP-cs-fixer tool.

I am not sure if we should override the Laminas coding standard rules here. Should we change the standard itself for new PHP versions?

Reply:
We can just suppress the rule. 

For example, see https://github.com/laminas/laminas-code/blob/dd19fe8e07cc3f374308565667eecd4958c22106/phpcs.xml#L38-L39

Reply:
(BTW, I will squash and merge to avoid having lots of "wip" commits on the history)